README.md

```diff
@@ -360,6 +360,40 @@ Notes:
 
 ...
 
+## Default values
+
+Default values are substituted in two cases:
+1. during flattening if the fields are not presented in the data
+2. during unflattening and schema evolution in case the target schema
+   has extra fields with the default values
+
+Notes:
+
+* Only zero-size arrays and maps are supported by now.
+* Default value may be inherited from an inner field with a default
+value or overridden. Example:
+```lua
+local schema = {
+    type = "record", name = "Frob", fields = {
+        { name = "foo", default = {f1=1, f2={f2_1=2}}, type =
```

Comment:
I think we should follow our code style consistently: whitespaces around `=` in tables.

Reply:
Rewritten.